app/uk/gov/hmrc/nationalinsurancerecord/config/ApplicationConfig.scala

```diff
@@ -18,12 +18,18 @@ package uk.gov.hmrc.nationalinsurancerecord.config
 
 import com.google.inject.{Inject, Singleton}
 import play.api.Configuration
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 
 @Singleton
-class ApplicationConfig @Inject()(configuration: Configuration) {
-  val responseCacheTTL = configuration.getInt("mongodb.responseTTL").getOrElse(throw new RuntimeException("MongoDB TTL is not configured"))
+class ApplicationConfig @Inject()(configuration: Configuration, servicesConfig: ServicesConfig) {
 
-  lazy val authorization: String = s"Bearer ${configuration.getString("microservice.services.des-hod.authorizationToken").getOrElse("Local")}"
+  val serviceUrl = servicesConfig.baseUrl("citizen-details")
+  val desUrl = servicesConfig.baseUrl("des-hod")
+  val authUrl = servicesConfig.baseUrl("auth")
 
-  lazy val desEnvironment: String = configuration.getString("microservice.services.des-hod.environment").getOrElse("Local")
+  val responseCacheTTL = configuration.get[Option[Int]]("mongodb.responseTTL").getOrElse(throw new RuntimeException("MongoDB TTL is not configured"))
+
+  val authorization: String = s"Bearer ${configuration.get[Option[String]]("microservice.services.des-hod.authorizationToken").getOrElse("Local")}"
```

Comment:
Suggestion

Because this is using microservice.services as the root property key you could use servicesConfig.getConfString

Reply:
Replace configuration.gets with the serviceConfig getter methods and removed the import and injection for configuration